src/js/tech/html5.js

```diff
@@ -1553,6 +1574,30 @@ Html5.resetMediaElement = function(el) {
   };
 });
 
+// This setter is declared outside of the loop because there is no 'playsinline'
```

Comment:
We can just describe this in the PR instead, don't think it's necessary, especially if we move it into the class syntax.